packages/devtools_app/lib/src/analytics/constants.dart

```diff
@@ -158,47 +139,54 @@ String topicDocumentationButton(String topic) => '${topic}DocumentationButton';
 String topicDocumentationLink(String topic) => '${topic}DocumentationLink';
 
 /// Analytic time constants specific for memory screen.
-class MemoryTimeAnalytics {
+class MemoryTime {
   static const adaptSnapshot = 'adaptSnapshot';
+  static const calculateDiff = 'calculateDiff';
   static const updateValues = 'updateValues';
 }
 
 /// Analytic event constants specific for memory screen.
-class MemoryEventAnalytics {
+class MemoryEvent {
   static const gc = 'gc';
   static const settings = 'settings';
+  static const autoSnapshot = 'autoSnapshot';
 
+  static const chartLegend = 'chartLegend';
+  static const chartAndroid = 'chartAndroid';
```

Comment:
renaming the variable is fine, but we should not change the String. This will create a new event in our analytics and will prevent us from being able to look at historical data before this change.